### PR TITLE
Adds CI configuration

### PR DESCRIPTION
**Pull request by deiga**
With this change you'll get automatic tests for contributions.

(You will need to enable this repo on travis-ci.org and ci.appveyor.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # declarations-js
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/steelbrain/declarations-js.svg)](https://greenkeeper.io/)
+[![Build status](https://ci.appveyor.com/api/projects/status/ukq9x783vr6vlhl6?svg=true)](https://ci.appveyor.com/project/deiga/declarations-js)
```
Comment (steelbrain) [inline README.md]: Can you change these paths and the one below to point to my repository? Thanks

**Review by steelbrain — CHANGES_REQUESTED** [review]
Thank you for working on this 🎉 

I've enabled the CIs and mentioned the one small change before this is merged

**Comment by deiga** [comment]
@steelbrain Done :)

**Comment by steelbrain** [comment]
Thaaank You

**Review by steelbrain — APPROVED** [review]

**Pull request merged**